MetricsUploader/include/MetricsUploader/MetricsUploader.h

```diff
@@ -53,7 +53,7 @@ class MetricsUploader {
 
   // Send log events to the server using metrics_uploader.
   // Returns true on success and false otherwise.
-  bool SendLogEvent(OrbitLogEvent_LogEventType log_event_type);
+  bool SendLogEvent(OrbitLogEvent_LogEventType log_event_type, int64_t event_duration_ms = 0);
```

Comment:
What about using `std::chrono::milliseconds` instead of  `int64_t`?

Reply:
Changed to `std::chrono::milliseconds`, thanks!